src/rpc/net.cpp

```diff
@@ -303,6 +304,23 @@ static RPCHelpMan addnode()
         throw std::runtime_error(
             self.ToString());
     }
+    const std::string manual_name{ConnectionTypeAsString(ConnectionType::MANUAL)};
```

Comment:
nit: you could create `manual_name` and `manual_block_relay_name` after checking the connection type with "remove":
```diff
diff --git a/src/rpc/net.cpp b/src/rpc/net.cpp
index 6155af323..9d87d89e3 100644
--- a/src/rpc/net.cpp
+++ b/src/rpc/net.cpp
@@ -304,11 +304,14 @@ static RPCHelpMan addnode()
         throw std::runtime_error(
             self.ToString());
     }
-    const std::string manual_name{ConnectionTypeAsString(ConnectionType::MANUAL)};
-    const std::string manual_block_relay_name{ConnectionTypeAsString(ConnectionType::MANUAL_BLOCK_RELAY)};
+
     if (strCommand == "remove" && !request.params[2].isNull()) {
         throw JSONRPCError(RPC_INVALID_PARAMS, "Cannot specify a connection type with 'remove'");
     }
+
+    const std::string manual_name{ConnectionTypeAsString(ConnectionType::MANUAL)};
+    const std::string manual_block_relay_name{ConnectionTypeAsString(ConnectionType::MANUAL_BLOCK_RELAY)};
+
     std::string connection_type_name{manual_name};
     if (!request.params[2].isNull()) {
         connection_type_name = request.params[2].get_str();
```

Reply:
made obsolete by the next comment.